### PR TITLE
Have second player

### PR DESCRIPTION
**Pull request by whaley-ethan**

### CODE REVIEW

File: src/main/java/tic/tac/toe/Board.java [hunk 1]
```diff
@@ -4,26 +4,40 @@
 public class Board {
```
Comment (jchung722) [inline src/main/java/tic/tac/toe/Board.java:4]: I think this class is doing too much; `Board` class should only be responsible for anything related to `Board`

File: src/main/java/tic/tac/toe/Board.java [hunk 2]
```diff
@@ -4,26 +4,40 @@
...
   
+  public char getCurrentPlayerSymbol() {
+    return currentPlayerSymbol;
+  }
+
   public boolean isCellEmpty(int cell) {
     return (board[cell-1] == ' ');
   }
   
+  public boolean isGameOver() {
```
Comment (jchung722) [inline src/main/java/tic/tac/toe/Board.java]: I would rename this method; what is this method telling us about `Board`?

File: src/main/java/tic/tac/toe/Board.java (hunk 1 above)
Comment (whaley-ethan) [reply src/main/java/tic/tac/toe/Board.java:4]: cleaned it up, moved non-board related logic to game

File: src/main/java/tic/tac/toe/Board.java (hunk 2 above)
Comment (whaley-ethan) [reply src/main/java/tic/tac/toe/Board.java]: Renamed to isBoardFull

**Comment by whaley-ethan** [comment]
Game class is getting large. Next major refactor will be adding in player class and pulling some of the logic into that. probably beyond the scope of this PR though

**Review by jchung722 — APPROVED** [review]

**Pull request merged**